documentation/presentation/demo/steps-to-lode.js

```diff
@@ -0,0 +1,2 @@
+// import lode-three
+import * as lodeLoader from "@kreativwebdesign/lode-three";
```

Comment:
Just wondering: what's the purpose of this file? Will we document the different steps in here? If so, shouldn't we use a markdown file instead?